src/styles/LoginStyles.ts

```diff
@@ -1,29 +1,31 @@
 import styled from 'styled-components';
 import { Button, TextField, Typography, Link, Container } from '@material-ui/core';
 
-export const LoginContainer = styled(Container)`
-    margin-top: 4%;
+export const MainDiv = styled('div')`
+    margin-top: 80px;
+`;
+
+export const FieldDiv = styled('div')`
+
 `;
 
 export const Label = styled(Typography)`
     color: #828282;
     font-size: 12px;
     font-weight: bold;
-    text-align: left;
-    padding: 6px 0px;
-    margin-left: 8%;
-    display: block;
+    margin-top: -20px;
+    margin-left: -83%;
 `;
-  
+
 export const Field = styled(TextField)`
     background-color: #F8F8F8;
     outlined: false;
     border-radius: 15px;
-    width: 340px;
+    width: 95%;
```

Comment:
If you use the FieldDiv as a wrapper, which already centers itself on the screen at 95% width, this can be `100%` width